docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -10,7 +10,7 @@ Program                   Description                           Required
 ========================  ====================================  ================================  ===================================
 :doc:`GEOS <../geos>`     Geometry Engine Open Source           Yes                               3.7, 3.6, 3.5
 `PROJ.4`_                 Cartographic Projections library      Yes (PostgreSQL and SQLite only)  5.2, 5.1, 5.0, 4.x
-:doc:`GDAL <../gdal>`     Geospatial Data Abstraction Library   Yes                               2.4, 2.3, 2.2, 2.1, 2.0
+:doc:`GDAL <../gdal>`     Geospatial Data Abstraction Library   Yes                               3.0, 2.4, 2.3, 2.2, 2.1, 2.0
```

Comment:
[GDAL 3.1 was released recently](https://github.com/OSGeo/gdal/releases/tag/v3.1.0). I think we could list it here too.

Reply:
The [release notes for GDAL 3.0.0](https://github.com/OSGeo/gdal/releases/tag/v3.0.0) refer to PROJ 6. Do we also need to check versions of that? I note that [PROJ 7.0.1 is available](https://proj.org/news.html#release-notes). We should also change from the outdated "PROJ.4" naming to just "PROJ".

I guess while we're there, [GEOS 3.8.1 is also available](https://trac.osgeo.org/geos/#Download).

Reply:
I created a separate ticket-31569 for GEOS 3.8+ support.